config/locales/models/activity.en.yml

```diff
@@ -424,6 +424,11 @@ en:
         invalid_oda_eligibility: The ODA Eligibility option is invalid
         invalid_programme_status: The Programme status option is invalid,
         invalid_call_open_date: The Call Open Date option is in an invalid format
+        invalid_call_close_date: The Call Close Date option is in an invalid format
```

Comment:
Oh, I see, the Call Close string ended up in the ensuing commit. Up to you if you want to move it - I appreciate very, very much that you've split the big change into smaller commits, so I feel bad about asking to rewrite history to move one line. ;)

Reply:
Nah, it's fine. That commit would have resulted in tests failing too (`spec/i18n_spec.rb`) and I like to make sure every commit has a green test status, so that helps keep me honest 🙂